cmd/cmd_test.go

```diff
@@ -27,6 +29,7 @@ const cfgHomeKey = "EXERCISM_CONFIG_HOME"
 // 	Cmd:    myCmd,
 // 	InitFn: initMyCmd,
 // 	Args:   []string{"fakeapp", "mycommand", "arg1", "--flag", "value"},
+//  MockInput: "first-input\nsecond\n"
```

Comment:
Would you mind aligning this with the other arguments and adding a comma to the end of the line so that the syntax is valid?